lib/iiif_manifest/v3/manifest_builder.rb

```diff
@@ -29,8 +29,32 @@ def to_h
         @to_h ||= builders.new(work).apply(top_record)
       end
 
+      # Utility method to wrap the obj into a IIIF V3 compliant language map as needed.
+      def self.language_map(obj)
+        # self.valid_language_map?(obj) ? obj : self.transform(obj)
```

Comment:
It looks like this comment isn't needed anymore.  Can you please remove it?  Same for the commented out methods below.